WriteOutputAllCells.c

```diff
@@ -36,7 +36,8 @@ void WriteOutputAllCells::write_data(std::vector<out_data_struct*>& all_out_data
   std::vector<size_t> start4(start4Vals, start4Vals + 4), count4(count4Vals, count4Vals + 4);
```

Comment:
This entire file has mixed (tabs *and* spaces) whitespace, which is really obnoxious to read. Can you please fix this?